tests/portal-chooser/assert-private-a11y.py

```
@@ -0,0 +1,100 @@
+        desktop = pyatspi.Registry.getDesktop(0)
+    except Exception as exc:
+        print(f"cannot reach the accessibility bus: {exc}", file=sys.stderr)
+        return 2
+
+    names = []
+    for app in desktop:
+        if app is None:
+            continue
+        try:
+            names.append(app.name or "<unnamed>")
+        except Exception as exc:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)